examples/js/loaders/GLTFLoader.js

```diff
@@ -29,6 +29,10 @@ THREE.GLTFLoader = ( function () {
 
 			var loader = new THREE.FileLoader( scope.manager );
 
+			if ( this.path !== undefined ) {
+				loader.setPath( path );
+			}
+
```

Comment:
In Three.js code style you need empty lines after `if ( ... ) {` and before `}`.

    if ( this.path !== undefined ) {

        loader.setPath( path );

    }

Or oneline is also good.

    if ( this.path !== undefined ) loader.setPath( path ):